.github/workflows/ci.yml

```diff
@@ -103,7 +103,7 @@ jobs:
       - name: Setup python
         uses: actions/setup-python@v2
         with:
-          python-version: 3.7
+          python-version: 3.8
```

Comment:
I thought bumping this would execute the tests in `test_sagemaker.py` [here](https://github.com/bentoml/BentoML/blob/master/tests/deployment/sagemaker/test_sagemaker.py#L27-L29), but that didn't do much. I use v3.8.5 on my MacOS and the tests seem to run fine locally. Probably another Github Actions build machine issue.